curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646e3d8a42296d01b0304b18.md

```diff
@@ -0,0 +1,211 @@
+---
+id: 646e3d8a42296d01b0304b18
+title: Step 17
+challengeType: 0
+dashedName: step-17
+---
+
+# --description--
+
+When the user selects the `Year, Month, Day` option, then the date format should reflect that. 
+
+Add a `case` where the value is `yyyy-mm-dd`. Inside this `case`, assign the constant `formattedDate` to the constant named `currentDateParagraph` using the `textContent` property.
```

Comment:
We will need to include a code example on how to add cases since this is the first time a switch statement is introduced. 

You will also need to mention that campers should add a break statement

Reply:
added